app/models/concerns/date_helpers.rb

```diff
@@ -0,0 +1,14 @@
+module DateHelpers
+
+  private
+
+  def valid_date?(date)
+     Date.parse(date.to_s) and return true
```

Comment:
`return true if Date.parse(date.to_s)` is good but we need to remember that: `if (something falsy)` returns `false`

`!!Date.parse(date.to_s) ` is good too. I've seen it used many times.

How about `Date.parse(date.to_s)  ? true : false`